Dockerfile

```diff
@@ -1,11 +1,18 @@
-FROM python:3.8-slim-buster
+FROM python:3.8 AS builder
+
+RUN python -m venv /opt/venv
+ENV PATH=/opt/venv/bin:$PATH
 
-RUN mkdir -p /usr/src/app
-COPY requirements.txt /usr/src/app/requirements.txt
-COPY swagger_server /usr/src/app/swagger_server
 WORKDIR /usr/src/app
+COPY requirements.txt .
+RUN pip install -r requirements.txt
 
-RUN pip3 install --no-cache-dir -r requirements.txt
+FROM python:3.8-slim-buster
```

Comment:
What is the benefit of having multi-stage build here? I think we will be copying everything from the virtualenv directory.